tests/DockerCredentialGenerator.php

```diff
@@ -25,14 +25,18 @@ public function __construct($dockerName)
     /**
      * {@inheritdoc}
      */
-    public function generate($path)
+    public function generate($path, $binPath, $pimVersion)
     {
-        $command = sprintf('docker exec %s php %s/app/console pim:oauth-server:create-client -e prod', $this->dockerName, $path);
+        if ("1.7" === $pimVersion) {
```

Comment:
or `$label = '1.7' === $pimVersion ? '--label="PHP client credentials"' : 'PHP client credentials'`
and no need if else.